website/docs/basics/blocks-styles.md

```diff
@@ -316,30 +346,90 @@ If the `variable` key is not set in options, the fallback is to output the value
 **Manifest:**
 ```json
 {
-  "attributes": {
-    "typographyUse": {
-      "type": "boolean",
-      "default": true,
-      "variable": "boolean"
-    }
-  },
-  "options": {
-    "typographyUse": [
-      "somethingFalse",
-      "somethingTrue"
-    ]
-  }
+	"attributes": {
+		"typographyUse": {
+			"type": "boolean",
+			"default": true,
+			"variable": "boolean"
+		}
+	},
+	"options": {
+		"typographyUse": [
+			"none",
+			"block"
+		]
+	}
 }
 ```
 
 **Output:**
 ```css
---typography-size: somethingTrue
+--typography-use: block;
 ```
 
-### Boolean variables inside the Block editor
+## Custom variables
 
-If you want to show different variables on the frontend and inside the Block editor you can use add values to the `index` key in options. The third value represents the value if false and the fourth editor value if true. Fallback tree is same as in Complex variables.
+If you need to set up many variables based on a single attribute value you can use the `custom` key. This key will search through the `options` for the attribute key and output the CSS variables based on values. Each custom variable key must be an `object`. You will also get the original key in the output.
```

Comment:
```suggestion
If you need a more complex setup with multiple variables for a single attribute value you can use the `custom` key. It will output the CSS variables based on values defined in the `options`. Each custom variable key must be an `object`. The original key is also output.
```